samples/nrf9160/http_application_update/src/main.c

```diff
@@ -111,6 +112,22 @@ void fota_dl_handler(const struct fota_download_evt *evt)
 		/* Re-enable button callback */
 		gpio_pin_enable_callback(gpiob, DT_ALIAS_SW0_GPIOS_PIN);
 		break;
+	case FOTA_DOWNLOAD_EVT_ERASE_PENDING:
+		printk("FOTA_DOWNLOAD_EVT_ERASE_PENDING modem firmware erase is"
+		       " pending, reboot or disconnect the LTE link\n");
+		err = lte_lc_offline();
+		if (err != 0) {
+			printk("Failed to disconnect\n");
+		}
+		break;
+	case FOTA_DOWNLOAD_EVT_ERASE_DONE:
+		printk("FOTA_DOWNLOAD_EVT_ERASE_DONE, reconnecting the LTE"
+		       " link\n");
+		err = lte_lc_connect();
```

Comment:
Do you mean lte_lc_normal() here?

Reply:
That depends. If lte_lc_normal() will resume the LTE link even if you have moved to another cell and the previous link is dropped then yes.

If not I would keep lte_lc_connect() to ensure that you get a new link.